frontend/static/global.css

```diff
@@ -1,3 +1,5 @@
+@import "https://unpkg.com/chota@latest";
```

Comment:
Wil je dit niet anders doen ipv via een url, maar via een dependency in je node_modules? (ik kan css imports niet goed)

Reply:
Oja goeie, zal het even aanpassen. Misschien dat ik dat hele ding er later nog weer uithaal want ik gebruik het nu alleen voor die tabbar maar dat kan denk ik ook even makkelijk zonder.